helm-charts/stable/prow-control-plane/templates/config-ConfigMap.yaml

```diff
@@ -94,6 +94,10 @@ data:
         - do-not-merge/hold
         - do-not-merge/work-in-progress
         - do-not-merge/invalid-owners-file
+        {{- if eq (len .Values.repositories) 0 }}
+        repos: []
+      merge_method: {}
+        {{- else }}
```

Comment:
The indentation looks off here right?

Reply:
Also instead of the conditionals on the key, can we do it for the values themselves. That way, it can input an empty list to `repos` instead of keeping track of the `repos` key in both the if and else condition.

Reply:
The indentation looks weird because line 98 and 99 are on different indentation levels. Lines 97, 100 and 109 are all part of the one if-else-end block which is why I indented them the same amount. I wasn't really sure how else to indent them. Maybe breaking down the if-else into two will make it more readable. Something like this

```
        {{- if eq (len .Values.repositories) 0 }}
        repos: []
        {{- else }}
        repos:
        {{-  range $repo := .Values.repositories }}
          - {{ $repo.org }}/{{ $repo.name }}
        {{- end }}
        {{- end }}
      {{- if eq (len .Values.repositories) 0 }}  
      merge_method: {}
      {{- else }}
      merge_method:
      {{-  range $repo := .Values.repositories }}
        {{ $repo.org }}/{{ $repo.name }}: squash
      {{- end }}
      {{- end }}
    decorate_all_jobs: true
```

Reply:
+1 to breaking it into 2 if/els blocks for readability